crates/cast/tests/cli/main.rs

```diff
@@ -522,35 +621,45 @@ casttest!(receipt_revert_reason, |_prj, cmd| {
     let rpc = next_http_rpc_endpoint();
 
     // <https://etherscan.io/tx/0x44f2aaa351460c074f2cb1e5a9e28cbc7d83f33e425101d2de14331c7b7ec31e>
-    cmd.cast_fuse().args([
+    cmd.args([
         "receipt",
         "0x44f2aaa351460c074f2cb1e5a9e28cbc7d83f33e425101d2de14331c7b7ec31e",
         "--rpc-url",
         rpc.as_str(),
-    ]);
+    ])
+    .assert_success();
     let output = cmd.stdout_lossy();
     assert!(!output.contains("revertReason"));
```

Comment:
It is currently not possible to make a negative assertion like this with Snapbox I think

Reply:
the point is that the output is shown in full and any change fails the test